lib/get_order_command.rb

```diff
@@ -0,0 +1,21 @@
+require 'order'
+
+class GetOrderCommand
+  attr_reader :response
+
+  def initialize(message)
+    @user_message = message
+  end
+
+  def response?
+    response
+  end
+
+  def run()
+    user_message = @user_message.gsub("order: ", "")
+    user_id_meal_researched = user_message[/(?<=\<@)(\w+)(?=>)/]
```

Comment:
Ahah yeah - it take the content between <@ the_content >.
I guess we should find a way to be more explicite :)